### ISSUE TITLE
String with a comma

### ISSUE THREAD

**Issue by rodw1995** [niftylettuce/dotenv-parse-variables#3]
I have a token to access a third party API. This token contains a comma ",". In my .env file the token is saved as:

TOKEN=abcde,fghij

When I parse the .env variables with dotenv-parse-variables, the token is parsed as an array, e.g. 
['abcde', 'fghij'] But of course I just want a string. Is this possible? Also with an asterisk, I am not able to get the desired result. Can you please help me?

**Comment by niftylettuce** [comment]
@rodw1995 can you submit a pull request that adds support for this?  The asterisk only disables conversion of the variable type, e.g. `"true"` instead of `true`

**Comment by niftylettuce** [comment]
@rodw1995 perhaps encapsulating the token with quotes? e.g. `TOKEN="abcd,defg"`?

**Comment by rodw1995** [comment]
@niftylettuce encapsulating with quotes doesn't work neither. The check on line 24 of index.js: 
```javascript
value.toString().indexOf(',') === -1
```
is preventing the asterisk from doing no parsing if the value contains a comma. Is this check really needed?

**Comment by niftylettuce** [comment]
I think the better approach is to add a line of code that detects if a string is wrapped in quotes.  Do you think you could add that and a test for it as well @rodw1995?

**Comment by niftylettuce** [comment]
It'd be something like:

```js
if (value.toString().indexOf('"') === 0 && value.toString().lastIndexOf('"') === value.toString().length - 1) {
  // do something here
}
```

**Comment by rodw1995** [comment]
I'm not able to run the test (Windows). Is it possible you run it? It should work (I think) https://github.com/rodw1995/dotenv-parse-variables

**Comment by niftylettuce** [comment]
@rodw1995 see my comments in #4

**Comment by niftylettuce** [comment]
Closed via #4, see v0.1.0 in NPM
